roles/prereqs/libsemanagepython/meta/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+
+dependencies:
+  - role: prereqs/packages
```

Comment:
libsemanagepython & libselinuxpython are using a different method because I did want to know what were your thougts about the new approach.